tests/pyccel/test_pyccel.py

```diff
@@ -1189,3 +1189,12 @@ def test_time_execution_flag():
     assert 'Total' in result_lines[-2]
     for l in result_lines[1:-1]:
         assert ' : ' in l
+
+#------------------------------------------------------------------------------
+def test_char:
+    if __name__ == '__main__':
+        string = np.empty((2, 5), dtype='c')
```

Comment:
You will need to make changes in `pyccel.ast.numpyext` or `pyccel.ast.numpytypes` so that 'c' is recognised.